skore/tests/unit/sklearn/test_cross_validation.py

```diff
@@ -988,3 +988,21 @@ def test_cross_validation_timings(
         "Predict time test",
     ]
     assert timings.columns.tolist() == expected_columns
+
+
+class BrokenEstimator(BaseEstimator, ClassifierMixin):
```

Comment:
Mixin goes on the left for the inheritance to work properly.

```suggestion
class BrokenEstimator(ClassifierMixin, BaseEstimator):
```

Reply:
Apologies, my bad.
Resolved